js/sample-authentication-code/README.md

```diff
@@ -0,0 +1,29 @@
+# NODE JS Access Token Authentication Sample
+
+## Requirements
+
+- Node v18+
+- Updated config.json
+
+## Required Client Details
+
+The following values need to be updated in config/config.json:
```

Comment:
Do we want to include where to get this information?

Reply:
I have kept this readme pretty similar to what we have for the java samples: https://github.com/jpmorgan-payments/authentication/tree/main/java/sample-authentication-code 

My understanding is clients are provided these details during onboarding and know to use them for token generation.